### PR TITLE
Tenet Chapel 1B/1F/2F

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -2990,7 +2990,7 @@ QUEST_LV_0100_20150317_002985	Let's defeat the hostile Venucelos
 QUEST_LV_0100_20150317_002986	$Royal Mausoleum's Treasure
 QUEST_LV_0100_20150317_002987	Challenge the spell of the Royal Mausoleum
 QUEST_LV_0100_20150317_002988	Pass the challenge
-QUEST_LV_0100_20150317_002989	Come To Your Senses! (1)
+QUEST_LV_0100_20150317_002989	Get a Hold of Yourself! (1)
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Original is just fine as the meaning is more closer to the original.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
 QUEST_LV_0100_20150317_005109	Follower Tomas asked you to find Vaidas and Vaidutis at Tenet Chapel B1.
-QUEST_LV_0100_20150317_005110	Deactivate the barrier of Gesti
+QUEST_LV_0100_20150317_005110	Deactivate the barrier that Gesti created
 QUEST_LV_0100_20150317_005111	$Vaidas told you that the demons have made a magic barrier at the basement's central altar. Destroy the barrier and continue looking for Follower Vaidutis.
 QUEST_LV_0100_20150317_005112	Find Follower Vaidutis
 QUEST_LV_0100_20150317_005113	Removed the barrier that was blocking the way to the 1st floor and defeated the Cyclops. Look for Vaidutis, who ran away to the 1st floor.
 QUEST_LV_0100_20150317_005114	Defeat the Cyclops
 QUEST_LV_0100_20150317_005115	Talk to Follower Algis
-QUEST_LV_0100_20150317_005116	Algis was waiting for you in 2nd Floor. Talk to him and ask what to do. 
+QUEST_LV_0100_20150317_005116	Algis was waiting for you on the 2nd Floor. Talk to him and ask what to do.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
-QUEST_LV_0100_20150317_005122	Seized the Bell Tower. Talk to Algis. 
-QUEST_LV_0100_20150317_005123	Seized the Bell Tower. Ask Algis what to do next. 
-QUEST_LV_0100_20150317_005124	Gesti destroyed the barrier in Central Hall!
-QUEST_LV_0100_20150317_005125	Gesti destroyed the Sventove Central Altar and stole the crest of space. You can't get the revelation this way and you'll be killed if you meet Gesti. Talk to Algis on how to solve the situation. 
-QUEST_LV_0100_20150317_005126	Make a trap in the Central Hall
-QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to emasculate Gesti. Collect the pieces of altar and place it on the eight pillars of Central hall.
+QUEST_LV_0100_20150317_005121	Algis says that in order to enter the hidden sanctum, you need the Crest of Space, which is hidden in the Central Altar, but Gesti is roaming around the area and it is too dangerous. Occupy the Bell Tower, a good observation point.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
-QUEST_LV_0100_20150317_005122	Seized the Bell Tower. Talk to Algis. 
-QUEST_LV_0100_20150317_005123	Seized the Bell Tower. Ask Algis what to do next. 
-QUEST_LV_0100_20150317_005124	Gesti destroyed the barrier in Central Hall!
-QUEST_LV_0100_20150317_005125	Gesti destroyed the Sventove Central Altar and stole the crest of space. You can't get the revelation this way and you'll be killed if you meet Gesti. Talk to Algis on how to solve the situation. 
-QUEST_LV_0100_20150317_005126	Make a trap in the Central Hall
-QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to emasculate Gesti. Collect the pieces of altar and place it on the eight pillars of Central hall.
+QUEST_LV_0100_20150317_005121	Algis says that in order to enter the hidden sanctum, you need the Crest of Space, which is hidden in the Central Altar, but Gesti is roaming around the area and it is too dangerous. Occupy the Bell Tower, a good observation point. 
+QUEST_LV_0100_20150317_005122	Occupied the Bell Tower. Talk to Algis.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
-QUEST_LV_0100_20150317_005122	Seized the Bell Tower. Talk to Algis. 
-QUEST_LV_0100_20150317_005123	Seized the Bell Tower. Ask Algis what to do next. 
-QUEST_LV_0100_20150317_005124	Gesti destroyed the barrier in Central Hall!
-QUEST_LV_0100_20150317_005125	Gesti destroyed the Sventove Central Altar and stole the crest of space. You can't get the revelation this way and you'll be killed if you meet Gesti. Talk to Algis on how to solve the situation. 
-QUEST_LV_0100_20150317_005126	Make a trap in the Central Hall
-QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to emasculate Gesti. Collect the pieces of altar and place it on the eight pillars of Central hall.
+QUEST_LV_0100_20150317_005121	Algis says that in order to enter the hidden sanctum, you need the Crest of Space, which is hidden in the Central Altar, but Gesti is roaming around the area and it is too dangerous. Occupy the Bell Tower, a good observation point. 
+QUEST_LV_0100_20150317_005122	Occupied the Bell Tower. Talk to Algis. 
+QUEST_LV_0100_20150317_005123	Occupied the Bell Tower. Ask Algis what to do next.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
-QUEST_LV_0100_20150317_005126	Make a trap in the Central Hall
-QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to emasculate Gesti. Collect the pieces of altar and place it on the eight pillars of Central hall.
+QUEST_LV_0100_20150317_005121	Algis says that in order to enter the hidden sanctum, you need the Crest of Space, which is hidden in the Central Altar, but Gesti is roaming around the area and it is too dangerous. Occupy the Bell Tower, a good observation point. 
+QUEST_LV_0100_20150317_005122	Occupied the Bell Tower. Talk to Algis. 
+QUEST_LV_0100_20150317_005123	Occupied the Bell Tower. Ask Algis what to do next. 
+QUEST_LV_0100_20150317_005124	Gesti destroyed the Sventove Central Altar!
+QUEST_LV_0100_20150317_005125	Gesti destroyed the Sventove Central Altar and stole the Crest of Space. You can't get the revelation this way and you'll be killed if you run into Gesti. Talk to Algis about how to solve the situation.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
-QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to emasculate Gesti. Collect the pieces of altar and place it on the eight pillars of Central hall.
+QUEST_LV_0100_20150317_005121	Algis says that in order to enter the hidden sanctum, you need the Crest of Space, which is hidden in the Central Altar, but Gesti is roaming around the area and it is too dangerous. Occupy the Bell Tower, a good observation point. 
+QUEST_LV_0100_20150317_005122	Occupied the Bell Tower. Talk to Algis. 
+QUEST_LV_0100_20150317_005123	Occupied the Bell Tower. Ask Algis what to do next. 
+QUEST_LV_0100_20150317_005124	Gesti destroyed the Sventove Central Altar!
+QUEST_LV_0100_20150317_005125	Gesti destroyed the Sventove Central Altar and stole the Crest of Space. You can't get the revelation this way and you'll be killed if you run into Gesti. Talk to Algis about how to solve the situation. 
+QUEST_LV_0100_20150317_005126	Make a trap at Sventove Central Hall
+QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to restrict Gesti. Collect the pieces of altar and place them on the eight pillars of Sventove Central Hall.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
 QUEST_LV_0100_20150317_005128	Placed the altar pieces on the eight pillars. Return to Algis.
 QUEST_LV_0100_20150317_005129	Algis is waiting for the Revelator on the 2nd floor of the Tenet Chapel. 
 QUEST_LV_0100_20150317_005130	Algis says he can't handle watching the place alone and asked you to use the power of altar to stamp demons with its power and defeat them.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
+QUEST_LV_0100_20150317_005121	Algis says that in order to enter the hidden sanctum, you need the Crest of Space, which is hidden in the Central Altar, but Gesti is roaming around the area and it is too dangerous. Occupy the Bell Tower, a good observation point. 
+QUEST_LV_0100_20150317_005122	Occupied the Bell Tower. Talk to Algis. 
+QUEST_LV_0100_20150317_005123	Occupied the Bell Tower. Ask Algis what to do next. 
+QUEST_LV_0100_20150317_005124	Gesti destroyed the Sventove Central Altar!
+QUEST_LV_0100_20150317_005125	Gesti destroyed the Sventove Central Altar and stole the Crest of Space. You can't get the revelation this way and you'll be killed if you run into Gesti. Talk to Algis about how to solve the situation. 
+QUEST_LV_0100_20150317_005126	Make a trap at Sventove Central Hall
+QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to restrict Gesti. Collect the pieces of altar and place them on the eight pillars of Sventove Central Hall.
 QUEST_LV_0100_20150317_005128	Placed the altar pieces on the eight pillars. Return to Algis.
 QUEST_LV_0100_20150317_005129	Algis is waiting for the Revelator on the 2nd floor of the Tenet Chapel.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
+QUEST_LV_0100_20150317_005121	Algis says that in order to enter the hidden sanctum, you need the Crest of Space, which is hidden in the Central Altar, but Gesti is roaming around the area and it is too dangerous. Occupy the Bell Tower, a good observation point. 
+QUEST_LV_0100_20150317_005122	Occupied the Bell Tower. Talk to Algis. 
+QUEST_LV_0100_20150317_005123	Occupied the Bell Tower. Ask Algis what to do next. 
+QUEST_LV_0100_20150317_005124	Gesti destroyed the Sventove Central Altar!
+QUEST_LV_0100_20150317_005125	Gesti destroyed the Sventove Central Altar and stole the Crest of Space. You can't get the revelation this way and you'll be killed if you run into Gesti. Talk to Algis about how to solve the situation. 
+QUEST_LV_0100_20150317_005126	Make a trap at Sventove Central Hall
+QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to restrict Gesti. Collect the pieces of altar and place them on the eight pillars of Sventove Central Hall.
 QUEST_LV_0100_20150317_005128	Placed the altar pieces on the eight pillars. Return to Algis.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
 QUEST_LV_0100_20150317_005129	Algis is waiting for the Revelator on the 2nd floor of the Tenet Chapel. 
 QUEST_LV_0100_20150317_005130	Algis says he can't handle watching the place alone and asked you to use the power of altar to stamp demons with its power and defeat them.
 QUEST_LV_0100_20150317_005131	Got rid of the demons and activated the altar. Return to Algis.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 12]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
 QUEST_LV_0100_20150317_005132	Charge the low level spirit crystal
-QUEST_LV_0100_20150317_005133	Algis says you need to defeat the demons following us. Use the Auka altar to defeat the demons and prevent them from chasing you. 
+QUEST_LV_0100_20150317_005133	Algis says you need to defeat the demons following us. Use the Auka Altar to defeat the demons and prevent them from chasing you.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
+QUEST_LV_0100_20150317_005127	Algis suggested using the pieces of destroyed altar to restrict Gesti. Collect the pieces of altar and place them on the eight pillars of Sventove Central Hall.
 QUEST_LV_0100_20150317_005128	Placed the altar pieces on the eight pillars. Return to Algis.
 QUEST_LV_0100_20150317_005129	Algis is waiting for the Revelator on the 2nd floor of the Tenet Chapel. 
 QUEST_LV_0100_20150317_005130	Algis says he can't handle watching the place alone and asked you to use the power of altar to stamp demons with its power and defeat them.
 QUEST_LV_0100_20150317_005131	Got rid of the demons and activated the altar. Return to Algis.
 QUEST_LV_0100_20150317_005132	Charge the low level spirit crystal
-QUEST_LV_0100_20150317_005133	Algis says you need to defeat the demons following us. Use the Auka altar to defeat the demons and prevent them from chasing you. 
+QUEST_LV_0100_20150317_005133	Algis says you need to defeat the demons following us. Use the Auka Altar to defeat the demons and prevent them from chasing you. 
 QUEST_LV_0100_20150317_005134	Activated the altar and shifted the demons' interest to the altar. Return to Algis.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 14]
```diff
@@ -5111,30 +5111,30 @@ QUEST_LV_0100_20150317_005106	Defeat Glizardons
...
 QUEST_LV_0100_20150317_005135	Defeat Egnome.
 QUEST_LV_0100_20150317_005136	Algis asked you to defeat the Egnome going around the chape.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 15]
```diff
@@ -5144,8 +5144,8 @@ QUEST_LV_0100_20150317_005139	Fight with Gesti
 QUEST_LV_0100_20150317_005140	Weakened Gesti with the power of central altar and eight pillars. Now you can fight on with Gesti. Emasculate Gesti while Algis prepares the spiritual crystal.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 16]
```diff
@@ -5144,8 +5144,8 @@ QUEST_LV_0100_20150317_005139	Fight with Gesti
...
 QUEST_LV_0100_20150317_005141	Failed to defeat Gesti but she ran away with serious injury. Talk to Algis.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

File: QUEST_LV_0100.tsv [hunk 17]
```diff
@@ -5144,8 +5144,8 @@ QUEST_LV_0100_20150317_005139	Fight with Gesti
 QUEST_LV_0100_20150317_005140	Weakened Gesti with the power of central altar and eight pillars. Now you can fight on with Gesti. Emasculate Gesti while Algis prepares the spiritual crystal.
 QUEST_LV_0100_20150317_005141	Failed to defeat Gesti but she ran away with serious injury. Talk to Algis. 
 QUEST_LV_0100_20150317_005142	Gesti got away, but there are still more things left to do in the chapel. Talk to Algis again.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Follower Algis

**Comment by Sunaries** [comment]
After changing Algis -> Follower Algis (since most of these, if not all, are just mentioned in the quest descriptions), this is good to merge. :+1:

**Comment by terrapin47** [comment]
Resolved conflicts.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**